### PR TITLE
Changed crt size check to only include NPM packaged files

### PR DESCRIPTION
**Pull request by sbSteveK**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .builder/actions/crt_size_check.py [hunk 1]
```diff
@@ -9,33 +9,29 @@
...
+        for root, dirs, files in os.walk(os.path.join(env.project.path, 'dist/native/')):
 
             for f in files:
                 fp = os.path.join(root, f)
-                current_folder_size += os.path.getsize(fp)
+                total_size += os.path.getsize(fp)
+
+        print(f"/native files total size: {total_size} bytes")
 
+        for root, dirs, files in os.walk(os.path.join(env.project.path, 'dist/')):
```
Comment (TingDaoK) [inline .builder/actions/crt_size_check.py:24]: you don't need a separate for loop here, the `.node` file will always be in `dist/`

File: .builder/actions/crt_size_check.py [hunk 2]
```diff
@@ -9,33 +9,29 @@
...
         # Maximum dist folder size in bytes
-        max_dist_size = 50_000_000
-        # Size of current folder
-        current_folder_size = 0
-        # Total size of files in dist folder
         total_size = 0
 
-        for root, dirs, files in os.walk(os.path.join(env.project.path, 'dist/')):
-            current_folder_size = 0
+        for root, dirs, files in os.walk(os.path.join(env.project.path, 'dist/native/')):
```
Comment (TingDaoK) [inline .builder/actions/crt_size_check.py:16]: I was wrong, maybe just leave `lib` and `include` out, which are C code and libraries that will not be included in the package we are shipping to npm

File: .builder/actions/crt_size_check.py [hunk 3]
```diff
@@ -9,33 +9,29 @@
...
                 fp = os.path.join(root, f)
-                current_folder_size += os.path.getsize(fp)
+                total_size += os.path.getsize(fp)
+
+        print(f"/native files total size: {total_size} bytes")
 
+        for root, dirs, files in os.walk(os.path.join(env.project.path, 'dist/')):
+            for f in files:
                 if 'aws-crt-nodejs.node' == f:
+                    fp = os.path.join(root, f)
```
Comment (justinboswell) [inline .builder/actions/crt_size_check.py:27]: what about the .js files in here? Those are also distributed

Comment (sbSteveK) [reply .builder/actions/crt_size_check.py:27]: Just spoke with Dengke. Going to change it to include all files in the dist folder except the ones in lib and include.

Comment (justinboswell) [reply .builder/actions/crt_size_check.py:27]: actually, come to think of it, everything in dist/ is shipped

**Review by justinboswell — APPROVED** [review]

**Pull request merged**